src/window.py

```diff
@@ -327,16 +327,36 @@ def on_preferences(self, *args):
         self.prefs_window.present()
 
     def on_about(self, *args):
-        dialog = Gtk.AboutDialog(transient_for=self)
-        dialog.set_logo_icon_name('com.github.huluti.Curtail')
-        dialog.set_program_name('Curtail')
-        dialog.set_version('1.4.0')
-        dialog.set_website('https://github.com/Huluti/Curtail')
-        dialog.set_authors(['Hugo Posnic', 'Steven Teskey', 'Andrey Kozlovskiy', 'Balló György', 'olokelo', 'Archisman Panigrahi'])
-        dialog.set_translator_credits(_("translator-credits"))
-        dialog.set_comments(_("Compress your images"))
-        dialog.set_license_type(Gtk.License.GPL_3_0)
-        dialog.present()
+        about = Adw.AboutWindow(
+                    transient_for=self,
+                    application_name='Curtail',
+                    application_icon='com.github.huluti.Curtail',
+                    developer_name='Hugo Posnic',
+                    license_type=Gtk.License.GPL_3_0,
+                    website='https://github.com/Huluti/Curtail',
+                    issue_url='https://github.com/Huluti/Curtail/issues',
+                    version='1.4.0',
+                    developers=[
+                        'Hugo Posnic https://github.com/Huluti'
+                    ],
+                    artists=[
+                        'Jakub Steiner https://github.com/jimmac',
+                    ],
+                    translator_credits=_("translator-credits"),
+                    copyright='© Hugo Posnic'
+                )
+        about.add_credit_section(
```

Comment:
These generally go in developers, prob tobias should go in designers

Reply:
I wasn't sure if he wanted to be in the "Design by" section as it's not yet perfect 😅 @bertob what do you think?